tools/gromacs/rmsd.xml

```diff
@@ -0,0 +1,150 @@
+                <option value="Yes">Yes</option>
+            </param>
+            <when value="Yes">
+                <param name="traj_input2" type="data" format="trr,xtc" label="Additional trajectory file" help="In XTC or TRR format."/>
+            </when>
+            <when value="No">
+            </when>
+        </conditional>
+        <param name="structure_input" type="data" format="tpr,pdb,gro" label="Structure file" help="In TPR, PDB, or GRO format. Use a tpr file if your system contains nonstandard atom types, as atomic masses may not be directly obtained from PDB or GRO files."/>
+        <param name="ndx_input" type="data" format="ndx" label="Index (NDX) file" help="Index file containing your system's different atomic and molecular groups."/>
+        <param name="index_rmsdgroup" value="0" min="0" type="integer" label="Index - RMSD group" help="Index of RMSD group - for proteins, this can generally be the same as the least squares fitting group, but can be different depending on the application."/>
+        <conditional name="fit">
+            <param name="fit_type" type="select" label="Fit Type" help="controls the least-squares fitting of the structures on top of each other: complete fit (rotation and translation), translation only, or no fitting at all.">
+                <option value="rot+trans">Rotation and translation. This is the default way.</option>
+                <option value="translation">Translation only.</option>
+                <option value="none">No fitting.</option>
+            </param>
+            <when value="rot+trans">
+                <param name="index_refgroup" value="0" min="0" type="integer" label="Index - least squares fitting group" help="Index number of the group you are fitting against, i.e. the reference group. For proteins, this group is commonly Backbone or C-alpha."/>
+            </when>
+            <when value="translation">
+                <param name="index_refgroup" value="0" min="0" type="integer" label="Index - least squares fitting group" help="Index number of the group you are fitting against, i.e. the reference group. For proteins, this group can be Backbone or C-alpha."/>
+            </when>
+            <when value="none">
+            </when>
+        </conditional>
+        <param name="outputformat" type="select" label="Output format">
+            <option value="xmgrace">Xmgrace. This is the default way.</option>
+            <option value="none">Raw Data.</option>
+        </param>
+        <param name="mir" type="select" label="Compare results with mirror image of the reference structure?" help="This is useful as a reference for assesing ‘significant’ RMSD values.">
+            <option value="false">No</option>
+            <option value="true">Yes</option>
+        </param>
+        <expand macro="log"/>
+    </inputs>
+    <outputs>
+        <data name="output1" format="xvg" from_work_dir="rmsd.xvg" label="GROMACS calculation of RMSD on ${on_string}"/>
```

Comment:
instead of output1/2 can we use more meaningful names. Those names are exposed into a few cases and help if they are meaningful.